src/components/CountrySelector/index.tsx

```diff
@@ -78,13 +88,28 @@ class CountrySelection extends Component {
     populateResults(filteredResults)
   }
 
+  handleMenuMouseClick = (event: Event) => {
+    const target = event.target as HTMLUListElement
+    // Intercept mouse click if event target is the displayed menu, i.e. scrollbar area
+    // (mouse clicks in on the menu list options the target will be a different class)
+    // Otherwise accessible-autocomplete picks up a mouse click on scrollbar area as a confirm event
+    if (
+      target.className.includes(
+        'onfido-sdk-ui-CountrySelector-custom__menu--visible'
+      )
+    ) {
+      event.preventDefault()
+    }
+  }
+
   componentDidMount() {
     if (this.props.idDocumentIssuingCountry) {
       this.props.actions.resetIdDocumentIssuingCountry()
     }
+    document.addEventListener('mousedown', this.handleMenuMouseClick)
```

Comment:
It would be good to remove this event listener on `componentWillUnmount`

Reply:
Good point. Forgot to do this clean up.